The big-endian constant was simply wrong; it should have been
the reverse of the little-endian constant.

--- gst-plugins-base-1.14.4/tests/check/libs/video.c
+++ gst-plugins-base-1.14.4/tests/check/libs/video.c
@@ -2659,7 +2659,7 @@ test_overlay_blend_rect_verify (gint x, gint y, gint width, gint height,
     fail_unless_equals_int (data[i + 3], 0x80);
 #else
     /* A - R - G - B */
-    fail_unless_equals_int (data[i], 0x00);
+    fail_unless_equals_int (data[i], 0x80);
     fail_unless_equals_int (data[i + 1], 0x80);
     fail_unless_equals_int (data[i + 2], 0x80);
     fail_unless_equals_int (data[i + 3], 0x80);
